ipaserver/plugins/ca.py

```diff
@@ -138,6 +138,19 @@ class ca(LDAPObject):
             ],
             'default_privileges': {'CA Administrator'},
         },
+        'System: Grant add on CAS container': {
```

Comment:
I prefer to keep consistency with other privileges, please call it 'System: Grant add on CAs container'